pkg/controller/namespace/namespace_controller.go

```diff
@@ -133,7 +133,8 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 	// Bind to workspace if the namespace created by kubesphere
 	_, hasWorkspaceLabel := namespace.Labels[tenantv1alpha1.WorkspaceLabel]
-	if hasWorkspaceLabel {
+	kubefedManaged := namespace.Labels[constants.KubefedManagedLabel] == "true"
```

Comment:
I think we should add some comments under this line. Just like:
// if the namespace doesn't have a label like `kubefed.io/managed: "true"`(single cluster environment) or it has a label like `kubefed.io/managed: "false"`(multi-cluster environment), we set the owner reference filed. Otherwise, we don't set the owner reference filed.

Reply:
done